src/components/TeacherViews/Onboarding/RoleSelection/RoleSelection.view.tsx

```diff
@@ -0,0 +1,92 @@
+import { zodResolver } from "@hookform/resolvers/zod";
+import { OakRadioButton, OakRadioGroup } from "@oaknational/oak-components";
+import { Control, UseFormTrigger, useForm } from "react-hook-form";
+
+import FieldError from "@/components/SharedComponents/FieldError";
+import Input from "@/components/SharedComponents/Input";
+import OnboardingForm from "@/components/TeacherComponents/OnboardingForm/OnboardingForm";
+import {
+  OnboardingFormProps,
+  RoleSelectFormProps,
+  roleSelectFormSchema,
+} from "@/components/TeacherComponents/OnboardingForm/OnboardingForm.schema";
+
+const roleOptions: Record<string, string> = {
+  "teacher-training": "Training to become a teacher",
+  "teacher-trainer": "Teacher trainer",
+  "private-tutor": "Private tutor",
+  "adult-helping-child":
+    "Adult helping a child, e.g. with revision, homeschooling",
+  "mat-or-lea": "Working at multi-academy trust or local educational authority",
+  nonprofit: "Working at an educational nonprofit",
+  other: "Other",
+};
```

Comment:
![ackroydperfect](https://github.com/user-attachments/assets/78fc1187-238f-4700-8a4a-be5f51a3ec4f)

Reply:
not sure how to interpret this 😂

Reply:
it's good! I meant to suggest mapping the roles to keys and only storing the key in planning. The copy will probably change 😉